src/App.jsx

```diff
@@ -1,22 +1,42 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
  import TodoList from './TodoList';
  import AddTodoForm from './AddTodoForm';
+ 
+ function App() {
+  const [todoList, setTodoList] = React.useState(([]));
+  const[isLoading, setIsLoading] = React.useState(true); 
+  
+  React.useEffect (() => {
```

Comment:
nit: now that you're importing `useEffect` above, you should just use `useEffect` here instead of `React.useEffect`. Same for other instances